src/redux/store.js

```diff
@@ -3,17 +3,12 @@ import ReduxLogger from 'redux-logger';
 
 import rootReducer from './reducers';
 import localStorageUtil from '../utils/localStorage';
+import thunk from 'redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
```

Comment:
ReduxDevtoolsExtension

Reply:
I know it is sometimes ugly, but it's a good rule to follow when project gets big

Reply:
It just takes more memory due to importing the whole package